src/repl/scala/tools/nsc/interpreter/LoopCommands.scala

```diff
@@ -24,40 +24,110 @@ class ProcessResult(val line: String) {
   override def toString = "`%s` (%d lines, exit %d)".format(line, buffer.size, exitCode)
 }
 
-trait LoopCommands {
+trait LoopCommands { self: { def echo(msg: String): Unit } =>
```

Comment:
Nominally, yes. I don't remember coding that, but it's probably not a hot path. If computers don't let you be lazy when it doesn't matter, then what are computers good for? I mean besides porn, online bullying and the surveillance state? Somewhere I just read the phrase, "surveillance capitalism" and it all made sense.